package.json

```diff
@@ -32,7 +38,7 @@
     "pods-install": "yarn example pod-install",
     "pods-update": "cd example/ios && pod update --silent",
     "bootstrap": "yarn && yarn example && yarn pods && yarn brew",
```

Comment:
Did you avoid migrating those commands? I had a lot more changes when I tried a couple months ago:
https://github.com/bitmovin/bitmovin-player-react-native/pull/193/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R35

Reply:
Yes I avoided those. My strategy was to do minimal changes step by step and see what is needed and what not. Since everything seems to be working I guess we are fine, but I'll double check this before merging.